tests/sentry/workflow_engine/handlers/action/test_notification_action.py

```diff
@@ -118,3 +119,41 @@ def test_handle_workflow_action_general_exception(self, mock_logger, mock_regist
             "Error invoking issue alert handler",
             extra={"action_id": self.action.id},
         )
+
+
+class TestMetricAlertRegistryInvoker(BaseWorkflowTest):
+    def setUp(self):
+        super().setUp()
+        self.project = self.create_project()
+        self.detector = self.create_detector(project=self.project)
+        self.action = Action(type=Action.Type.DISCORD)
+        self.group, self.event, self.group_event = self.create_group_event()
+        self.job = WorkflowJob({"event": self.group_event})
+
+    @mock.patch(
```

Comment:
nit: if you are sharing a patch between all functions in a test class, you can move the patch decorator above the class declaration on line 124